doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -30,6 +30,55 @@ If multiple drivers are enabled, the first ordered item in this table takes prec
 
 Enabling or disabling PSA driver specific configurations controls the support for a given algorithm, per driver.
 
+Key type configuration
+**********************
+
+To enable block or stream cipher algorithms, set one or more of the following Kconfig options:
```

Comment:
Shall we just say "cryptographic algorithms", as then it includes asymmetric crypto, and the key agreement algorithms as well
```suggestion
To enable cryptographic algorithms, set one or more of the following Kconfig options:
```